### PR TITLE
Javascript error when there are no deployment steps

### PR DESCRIPTION
**Pull request by lampersky**
fixes https://github.com/OrchardCMS/OrchardCore/issues/15301

/cc: @Piedone

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.Deployment/Assets/js/steporder.js [hunk 1]
```diff
@@ -19,13 +19,14 @@ function updateStepOrders(oldIndex, newIndex) {
...
-
-    var sortable = Sortable.create(sortable, {
-        handle: ".ui-sortable-handle",
-        onSort: function (evt) {
-            var oldIndex = evt.oldIndex;
-            var newIndex = evt.newIndex;
-            updateStepOrders(oldIndex, newIndex);
-        }
-    });
+    if (sortable) {
```
Comment (MikeAlhayek) [inline src/OrchardCore.Modules/OrchardCore.Deployment/Assets/js/steporder.js:22]: I like to return `if (!sortable)` to save indentation.

Also, please eliminate the need for jQuery here by replacing $(function()... to

`document.addEventHandler('DOMContentLoaded', function()`

**Review by MikeAlhayek — CHANGES_REQUESTED** [review]

File: src/OrchardCore.Modules/OrchardCore.Deployment/Assets/js/steporder.js (hunk 1 above)
Comment (Piedone) [reply src/OrchardCore.Modules/OrchardCore.Deployment/Assets/js/steporder.js:22]: I second the indentation, but as for jQuery: we already use it everywhere on the admin, and this script uses it 6 times too. I thus wouldn't change the ready function and risk breaking something in some obscure way when we wouldn't really get any benefit (and especially don't scope creep into trying to get rid of jQuery too).

**Review by Piedone — CHANGES_REQUESTED** [review]
Thank you for the quick fix!

**Review by Piedone — APPROVED** [review]

**Comment by Piedone** [comment]
When you're done, please re-request review, so reviewers can get notified:

image

In this case, doing this for Mike remains :).

**Comment by sebastienros** [comment]
@lampersky I am looking at this PR as it reflect another problem we saw with one of yours. It appears you don't know about the gulp pipeline we use to generate /wwwroot and minimized files and you updated these by hand which introduced errors. For instance the wwwroot version has the opposite test, and the minimized version was forgotten altogether. We'll fix it in main but it's important you understand how to get this command running when you update a javascript file from an Assets folder. https://docs.orchardcore.net/en/latest/docs/guides/gulp-pipeline/

**Comment by MikeAlhayek** [comment]
@lampersky the issue was fix by #15387

**Pull request merged**